documentatie/use-cases/GSB-subfuncties.md

```diff
@@ -53,3 +53,20 @@ __uitbreidingen__:
 
 - De eerste stap van invoer is aangeven of er herteld is vanwege een verschil tussen aantal toegelaten kiezers en aantal uitgebrachte stemmen. Hoe verhoudt de invoer van die stap zich tot het oplossen van waarschuwingen over aantallen toegelaten kiezers en uitgebrachte stemmen?
 - Als de coördinator het PV terugstuurt in het proces, naar welk punt dan precies?
+
+
+## De coördinator bewerkt de stembureaus
+
+__niveau:__ subfunctie, vis, 🐟
+
+### Hoofdscenario en uitbreidingen
+
+__hoofdscenario__:
+
+1. De coördinator voegt stembureaus toe, past stembureaus aan en verwijdert stembureaus waar nodig.
+2. De applicatie controleert dat de bewerking aan de validaties voldoet.
+
+__uitbreidingen__:
+2a. (bij verwijderen) Er is al invoer of een resultaat voor het stembureau:
```

Comment:
Nu ik dit zo lees; zouden we deze check niet ook bij 'bewerken' moeten doen? De naam of het nummer van een stembureau aanpassen als er al invoer is, lijkt me ook niet wenselijk.